OracleDatabase/SingleInstance/dockerfiles/19.3.0/configTcps.sh

```diff
@@ -109,6 +109,9 @@ function disable_tcps() {
 ################## MAIN ###################
 ###########################################
 
+ORACLE_SID="$(grep "$ORACLE_HOME" /etc/oratab | cut -d: -f1)"
```

Comment:
Possibly this would be better?
ORACLE_SID="${ORACLE_SID:=$(grep "$ORACLE_HOME" /etc/oratab | cut -d: -f1)}"

Reply:
This will create problem when ORACLE_SID is passed using -e option and it has small case alphabets. We need to put additional instruction to capitalise the ORACLE_SID.

`ORACLE_SID="$(grep "$ORACLE_HOME" /etc/oratab | cut -d: -f1)` will take care of this problem, as the in extracted ORACLE_SID from the oratab will already have capital alphabets.